src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,23 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger();
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Authentication failed for username 'bob'. Reason: {}", e.getMessage());
         }
+        logger.info("Attempting to complete order for user ID '{}'.",
+                user != null ? user.getUserId() : null);
```

Comment:
There is a potential `NullPointerException` risk here. You should ensure the `user` object is not null before calling `getUserId()`. Consider handling this more gracefully to avoid runtime exceptions.